corehq/apps/saved_reports/tests/test_models.py

```diff
@@ -160,7 +161,7 @@ def test_huge_reports_with_attachments_resend_only_attachments(self):
         self.mock_send_email.assert_called_with('Test Report',
             ['test1@dimagi.com', 'test2@dimagi.com'],
             'Unable to generate email report. Excel files are attached.',
-            email_from='commcarehq-noreply@example.com',
+            email_from=settings.DEFAULT_FROM_EMAIL,
```

Comment:
I think I would have preferred to see this done via the [override_settings](https://docs.djangoproject.com/en/4.0/topics/testing/tools/#django.test.modify_settings) context manager. A failed test is still going to be difficult to read, but I prefer to check for values that are specified within the test, than pulled from another file. Not blocking, though

Reply:
Fair point, although I don't have a strong opinion on which way is better and don't feel it's worth making another PR to change it. Feel free to make the PR yourself if you feel more strongly than I do.